packages/icons-react/documentation/IconsExample.tsx

```diff
@@ -1,79 +1,94 @@
+    InfoIcon,
+    ErrorIcon,
+    WarningIcon,
+    SuccessIcon,
 } from "../src";
 import { IconVariant } from "../src/icons/types";
 import { IconExample } from "./internal/IconExample";
 import { IconsExampleGrid } from "./internal/IconsExampleGrid";
 
-export const choiceProps = [
-    {
-        name: "Variant",
-        values: ["inherit", "small", "medium", "large"],
-        defaultValue: 1,
-    },
-    {
-        name: "Farge",
-        values: ["inherit", "feil", "suksess"],
-        defaultValue: 0,
-    },
-];
+export const iconsExampleKnobs: ExampleKnobsProps = {
+    boolProps: ["Bold"],
+    choiceProps: [
+        {
+            name: "Variant",
+            values: ["inherit", "medium", "small"],
+            defaultValue: 1,
+        },
+    ],
+};
 
-export const IconsExample: React.FC<ExampleComponentProps> = ({ choiceValues }) => {
+export const IconsExample: React.FC<ExampleComponentProps> = ({ choiceValues, boolValues }) => {
     const allIcons = [
-        Close,
-        CheckMark,
-        Plus,
-        Search,
-        Hamburger,
-        Calendar,
-        ArrowUpRight,
-        ArrowUp,
-        ArrowDown,
-        ArrowRight,
-        ArrowLeft,
-        Info,
-        Error,
-        Warning,
-        Success,
+        CloseIcon,
+        CheckIcon,
+        PlusIcon,
+        SearchIcon,
+        HamburgerIcon,
+        CalendarIcon,
+        ArrowNorthEastIcon,
+        ArrowUpIcon,
+        ArrowDownIcon,
+        ArrowRightIcon,
+        ArrowLeftIcon,
+        ChevronDownIcon,
+        ChevronLeftIcon,
+        ChevronRightIcon,
+        ChevronUpIcon,
+        CopyIcon,
+        DotsIcon,
+        QuestionIcon,
+        InfoIcon,
+        ErrorIcon,
+        WarningIcon,
+        SuccessIcon,
     ];
 
-    const colorValue = choiceValues ? choiceValues["Farge"] : "inherit";
-    const color = colorValue === "inherit" ? undefined : colorValue;
     const variant = choiceValues ? (choiceValues["Variant"] as IconVariant) : "small";
+    const bold = boolValues?.["Bold"] || false;
 
     const [fontSize, setFontSize] = useState("1rem");
 
     return (
         <div style={{ width: "100%" }}>
             {variant === "inherit" && (
-                <label>
-                    Sett fontstørrelse
-                    <input
-                        className="jkl-spacing-l--top jkl-spacing-l--bottom"
-                        value={fontSize}
-                        onChange={(e) => setFontSize(e.target.value)}
-                    />
-                </label>
+                <TextInput
+                    label="Fontstørrelse"
+                    className="jkl-spacing-24--bottom"
+                    value={fontSize}
+                    onChange={(e) => setFontSize(e.target.value)}
+                    width="10ch"
+                />
             )}
-            <IconsExampleGrid style={{ fontSize }} columns="four" color={color}>
+            <IconsExampleGrid style={{ fontSize }} columns="two">
```

Comment:
Jeg endret denne fra fire til to for å gjøre den penere i https://jokul.fremtind.no/profil/ikoner. Vi ble enige på mikrodemoen i går om å kutte ned på innholdet om ikoner på profilsiden og heller henvise til Icons under komponenter. Ønsket var å beholde fire kolonner her.

Reply:
OK, så tilbake til fire komponenter, og så fjerne eksemplet fra profilsiden?